extensions/sql-database-projects/package.json

```diff
@@ -120,8 +120,8 @@
         "title": "%sqlDatabaseProjects.importDatabase%",
         "category": "%sqlDatabaseProjects.displayName%",
           "icon": {
-            "dark": "images/dark/databaseProject.svg",
-            "light": "images/light/databaseProject.svg"
+            "dark": "images/databaseProjectToolbar.svg",
```

Comment:
I believe you can simplify this to:

`
"icon": "images/databaseProjectToolbar.svg"
`